packages/core/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt

```diff
@@ -129,7 +129,18 @@ class LottieAnimationViewPropertyManager(view: LottieAnimationView) {
         }
 
         sourceDotLottie?.let { assetName ->
-            var file = File(assetName)
+
+            var file: File? = null
+
+            val scheme = runCatching { Uri.parse(assetName).scheme }.getOrNull()
+            // if the asset path has file:// prefix, which indicates locally stored file, parse the path to be able to load it properly
+            // This is useful for apps, which are using OTA (CodePush, Expo-Updates etc.)
+            if (scheme == "file") {
+                val uri = Uri.parse(assetName)
+                file = File(uri.path!!)
```

Comment:
Can you do this without force unwraping? File is already null, you can just do `uri.path?.let { }`.

Reply:
And maybe log into Logcat something if File is still null after all the operations (uri.path somehow did not resolve to something)

Reply:
Same comment. Please fix this and it's good to go.

Reply:
Beautiful, approved and set the CI to run, can merge after it gives us the go ahead